kyverno-integration/modules/enforce/exceptions/crossplane.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kyverno.io/v2beta1
+kind: PolicyException
+metadata:
+  name: crossplane-system-cnoe-operation
+  namespace: kyverno
+spec:
+  exceptions:
```

Comment:
Can we install crossplane different to make it more align with best practice policies?

Reply:
Am not sure what you mean here can you elaborate

Reply:
This adds exceptions to allow crossplane to work, is there some changes we can do the crossplane installation [here](https://github.com/cnoe-io/stacks/tree/main/crossplane-integrations) to not need these many exceptions?

Reply:
I could, but since Stacks have no conditionals, if the exceptions are installed without Kyverno, the installation will fail as k8s won't know what object that is. Let me know.

Reply:
@nabuskey ^^ let me know if you have any thoughts

Reply:
I don't see a path forward for this without native conditionals and parameter sharing in packages. Best we can do is to make changes to the Crossplane stack but it's outside scope of this PR.